doc/dev/issues/resolve_issues_effectively.md

```diff
@@ -11,6 +11,8 @@ Most issues can be classified into 3 categories. They are
 ## Usage error (For users)
 If you are not familiar with the SDK usage of a service, you can find relevant examples in [this repo][sample repo] in most cases.
 
+For some common errors, you can check [here](## Summary Of Common Errors).
```

Comment:
the view does not take effect
![image](https://user-images.githubusercontent.com/70930885/171072440-7266416a-3622-4a9f-8843-94c48ea50c37.png)